.github/workflows/code-formatting.yml

```diff
@@ -7,6 +7,21 @@ on:  # Trigger the workflow on push or pull request, but only for the master bra
     branches: [master]
 
 jobs:
+  isort:
+    name: Check valid import formatting with isort
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install isort
+        run: pip install isort==5.6.4
+      - name: Run isort
+        run: isort --settings-path=./pyproject.toml --diff .
```

Comment:
Could you keep `--check` option to obtain UNIX exit status? Without `--check`, it always exits with `0`, and it means that `isort` check always passes on GitHub Actions no matter what the source code is. I have checked the behaviour at https://github.com/akihironitta/ci-example/runs/1305737153.

Reply:
Good catch. I assumed that `--diff` also caused returned an error code, since it doesn't modify the files in-place (much like `--check-only` in this sense). I didn't think to test it out. I've added back the `--check-only` flag.

Reply:
Thank you for the swift action!

Reply:
`isort --help` now has the description of the exit status of `--check` on the main branch! https://github.com/PyCQA/isort/pull/1590
```
Checks the file for unsorted / unformatted imports and 
prints them to the command line without modifying the file. 
Returns 0 when nothing would change and returns 1 when 
the file would be reformatted.
```